components/lvgl_esp32_drivers/lvgl_spi_conf.h

```diff
@@ -26,9 +26,11 @@ extern "C" {
 #endif
 #define DISP_SPI_CLK CONFIG_LVGL_DISP_SPI_CLK
 #if defined (CONFIG_LVGL_DISPLAY_USE_SPI_CS)
-#define DISP_SPI_CS CONFIG_LVGL_DISP_SPI_CS
+    #define DISP_SPI_CS CONFIG_LVGL_DISP_SPI_CS
+    #define DISP_SPI_INPUT_DELAY_NS CONFIG_LVGL_DISP_SPI_INPUT_DELAY_NS
```

Comment:
I've tested this PR targeting an IL9341 display controller and the `CONFIG_LVGL_DISP_SPI_INPUT_DELAY_NS` symbol isn't defined, so the build breaks. Can we replace the line `#define DISP_SPI_INPUT_DELAY_NS CONFIG_LVGL_DISP_SPI_INPUT_DELAY_NS` with something like this to avoid it?

```c
#if defined CONFIG_LVGL_DISP_SPI_INPUT_DELAY_NS
    #define DISP_SPI_INPUT_DELAY_NS CONFIG_LVGL_DISP_SPI_INPUT_DELAY_NS
#else
    #define DISP_SPI_INPUT_DELAY_NS 0
#endif
```

Reply:
Yeah, of course, I'll take a look at that.  Also, that isn't the right place to put the input delay anyway, so I'll fix that too.

Reply:
So if I disable the MISO pin for RA8875, I get that build error.  The issue is simply that I put that #define in the wrong place.  I just did a forced-push, so this should be fixed now.  See if it works for you now, and let me know if you would still prefer to add a if-not-defined default.  Thanks for catching that.

Reply:
Thanks for the quick reply, just tested this again and it builds.